wpilibc/src/main/native/cpp/DataLogManager.cpp

```diff
@@ -138,6 +140,12 @@ void Thread::Main() {
                      entry.path().string());
         }
       }
+    } else if (freeSpace < 2 * kFreeSpaceThreshold) {
+      fmt::print(stderr,
```

Comment:
This should match the java implementation and report to the driver station using `FRC_ReportError`

Reply:
What error code am I supposed to use?